packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page.tsx

```diff
@@ -53,7 +54,7 @@ const Page: NextPageWithLayout = () => {
   }
 
   if (!rootEntityAndSubgraph) {
-    return <h1>Entity not found</h1>;
+    return <PageErrorState />;
```

Comment:
Are you setting the status code to 404 here?

Reply:
Nope, I just showed the error status of the page, not redirected or anything, just rendered a react component instead of `h1` here.  Since the ticket reads "Add page entity-not-found state"